storage/config/template.go

```diff
@@ -26,9 +26,10 @@ func (store *templateStore) Create(ctx context.Context, tmpl *domain.Template) d
 }
 
 func (store *templateStore) GetByID(ctx context.Context, id string) (*domain.Template, domain.IError) {
-	var t = new(config.Template)
+	t := new(config.Template)
 
 	for _, v := range store.cfg.Templates {
+		v := v
```

Comment:
Although it is not currently affected (without the current fix) because of the nature of operations done inside the `for` statement, a slight change could result in unpredictable behavior if not used.